app/routes/events/components/EventEditor/EditorSection/Details.tsx

```diff
@@ -108,7 +108,8 @@ const Details: React.FC<Props> = ({ values }) => {
                 <Tooltip content="Åpne stedet i MazeMap">
                   <MazemapLink
                     mazemapPoi={values.mazemapPoi?.value}
-                    linkText="↗️"
+                    iconOnly
+                    style={{ position: 'relative', top: '0.8rem' }}
```

Comment:
What other places do we use this? Is there any reason why you haven't included this in the <MazemapLink/> component itself? Or added it using a css file and not inline styling?

Reply:
In meeting and event editor we want this styling, cause I couldn't figure out how to center it (centering inside the parent Flex is wrong, look at the first before pic), so it has to be offset like this, but not when used below the MazemapEmbed